frontend/src/properties/propertysections/TreePropertySection.types.ts

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { Subscriber, Tree } from 'form/Form.types';
+import { Selection } from 'workbench/Workbench.types';
+
+export interface TreePropertySectionProps {
+  editingContextId: string;
+  formId: string;
+  widget: Tree;
+  subscribers: Subscriber[];
+  setSelection: (selection: Selection) => void;
```

Comment:
It's a bit odd to ask for setSelection but not for the selection to highlight the current selection. It only works in one direction.